Unithereum.CodeGen/UnithereumCodeGenProcessor.cs

```diff
@@ -164,75 +211,105 @@ private static void Generate(string abiPath, string? binPath)
+                    "Failed to generate Nethereum contract service code: " + error
+                );
             }
-            
+
             AssetDatabase.ImportAsset(
                 Path.Combine(Path.GetFileName(Application.dataPath), "ContractServices"),
-                ImportAssetOptions.ImportRecursive);
+                ImportAssetOptions.ImportRecursive
+            );
         }
 
         private static string GetCodeGenNamespace()
         {
-            var productName =
-                SanitizeLeadingDots(
-                    SanitizeTrailingDots(
-                        string.Join("",
-                            Application.productName.Select(c =>
-                            {
-                                var cat = CharUnicodeInfo.GetUnicodeCategory(c);
-                                return cat == UnicodeCategory.UppercaseLetter ||
-                                       cat == UnicodeCategory.LowercaseLetter ||
-                                       cat == UnicodeCategory.TitlecaseLetter ||
-                                       cat == UnicodeCategory.ModifierLetter ||
-                                       cat == UnicodeCategory.OtherLetter ||
-                                       cat == UnicodeCategory.SpacingCombiningMark ||
-                                       cat == UnicodeCategory.DecimalDigitNumber ||
-                                       cat == UnicodeCategory.LetterNumber ||
-                                       cat == UnicodeCategory.ConnectorPunctuation ||
-                                       c == '.'
-                                    ? c.ToString()
-                                    : "_";
-                            }))));
+            var productName = SanitizeLeadingDots(
+                SanitizeTrailingDots(
+                    string.Join(
+                        "",
+                        Application.productName.Select(c =>
+                        {
+                            var cat = CharUnicodeInfo.GetUnicodeCategory(c);
+                            return
+                                cat == UnicodeCategory.UppercaseLetter
+                                || cat == UnicodeCategory.LowercaseLetter
+                                || cat == UnicodeCategory.TitlecaseLetter
+                                || cat == UnicodeCategory.ModifierLetter
+                                || cat == UnicodeCategory.OtherLetter
+                                || cat == UnicodeCategory.SpacingCombiningMark
+                                || cat == UnicodeCategory.DecimalDigitNumber
+                                || cat == UnicodeCategory.LetterNumber
+                                || cat == UnicodeCategory.ConnectorPunctuation
+                                || c == '.'
+                                ? c.ToString()
+                                : "_";
+                        })
+                    )
+                )
+            );
```

Comment:
I personally prefer the closing parenthesis to on the same line with the last parameter like so:

```suggestion
            var productName = SanitizeLeadingDots(
                SanitizeTrailingDots(
                    string.Join(
                        "",
                        Application.productName.Select(c =>
                        {
                            var cat = CharUnicodeInfo.GetUnicodeCategory(c);
                            return
                                cat == UnicodeCategory.UppercaseLetter
                                || cat == UnicodeCategory.LowercaseLetter
                                || cat == UnicodeCategory.TitlecaseLetter
                                || cat == UnicodeCategory.ModifierLetter
                                || cat == UnicodeCategory.OtherLetter
                                || cat == UnicodeCategory.SpacingCombiningMark
                                || cat == UnicodeCategory.DecimalDigitNumber
                                || cat == UnicodeCategory.LetterNumber
                                || cat == UnicodeCategory.ConnectorPunctuation
                                || c == '.'
                                ? c.ToString()
                                : "_";
                        }))));
```

But I'm not sure if this is customizable in `.editorconfig`. 🤔

Reply:
This is CSharpier behavior and it seems not customizable...